### PR TITLE
Introduce alias support for index pattern

### PR DESCRIPTION
**Pull request by ruflin**
The goal of this PR is that in case an alias is detected in the fields.yml, for the index pattern the fields from the original are copied over.

### CODE REVIEW

File: libbeat/kibana/testdata/beat-6.json [hunk 1]
```diff
@@ -2,8 +2,8 @@
...
+        "fieldFormatMap": "{\"long\":{\"id\":\"url\",\"params\":{\"inputFormat\":\"string\",\"labelTemplate\":\"long template\",\"outputFormat\":\"float\",\"outputPrecision\":5,\"urlTemplate\":\"_a=(query:(language:lucene,query:'context.app.name:\\\"{{value}}\\\"'))\"}},\"alias\":{\"id\":\"url\",\"params\":{\"inputFormat\":\"string\",\"labelTemplate\":\"long template\",\"outputFormat\":\"float\",\"outputPrecision\":5,\"urlTemplate\":\"_a=(query:(language:lucene,query:'context.app.name:\\\"{{value}}\\\"'))\"}}}",
```
Comment (ruflin) [inline libbeat/kibana/testdata/beat-6.json]: @simitt Is there a good way to generate these files? I'm fighting to get the tests passing.

Comment (ruflin) [inline libbeat/kibana/testdata/beat-6.json]: @simitt Is there a good way to generate these files? I'm fighting to get the tests passing.

Comment (simitt) [inline libbeat/kibana/testdata/beat-6.json]: @ruflin you were asking how you could make the `alias` show up in the `fieldFormat` section of the Kibana index pattern. You need to add the handling [here](https://github.com/elastic/beats/blob/master/libbeat/kibana/fields_transformer.go#L199).

Comment (ruflin) [reply libbeat/kibana/testdata/beat-6.json]: This is the way I make it show up fieldsFormat: https://github.com/elastic/beats/pull/10075/files#diff-086ab841dd8317d2aeec5107211984c2R87 Running it against Kibana this trick seems to work. My problem is with the tests and `fieldFormatMap`

**Comment by exekias** [comment]
I pushed a commit that I think fixes this, please take a look, I had to change part of the test data as I think something was missing

File: libbeat/kibana/testdata/beat-6.json (hunk 1 above)
Comment (simitt) [reply libbeat/kibana/testdata/beat-6.json]: Actually I don't recall a good way for creating the tests automatically.

File: libbeat/kibana/fields_transformer.go [hunk 2]
```diff
@@ -24,6 +24,8 @@ import (
 	"github.com/elastic/beats/libbeat/common"
 )
 
+var v640 = common.MustNewVersion("6.4.0")
```
Comment (ruflin) [inline libbeat/kibana/fields_transformer.go]: Almost forgot that part.

**Pull request merged**